tests/bootstrap.php

```diff
@@ -26,12 +34,13 @@
 }
 
 require dirname( dirname( __FILE__ ) ) . '/tests/stripe-http-client-mock.php';
-require dirname( dirname( __FILE__ ) ) . '/inc/css/class-css-utility.php';
-require dirname( dirname( __FILE__ ) ) . '/inc/plugins/class-block-conditions.php';
-require dirname( dirname( __FILE__ ) ) . '/inc/plugins/class-dynamic-content.php';
-require dirname( dirname( __FILE__ ) ) . '/plugins/otter-pro/inc/plugins/class-block-conditions.php';
-require dirname( dirname( __FILE__ ) ) . '/plugins/otter-pro/inc/plugins/class-dynamic-content.php';
+//require dirname( dirname( __FILE__ ) ) . '/inc/css/class-css-utility.php';
+//require dirname( dirname( __FILE__ ) ) . '/inc/plugins/class-block-conditions.php';
+//require dirname( dirname( __FILE__ ) ) . '/inc/plugins/class-dynamic-content.php';
+//require dirname( dirname( __FILE__ ) ) . '/plugins/otter-pro/inc/plugins/class-block-conditions.php';
```

Comment:
Here I'm not sure if commenting this was a typo from testing ? Apart from this everything is good.